### PR TITLE
Adds exercise 2

### PR DESCRIPTION
**Pull request by vincentaudebert**
@thibaudcolas to review ❤️

See instructions once built or directly in https://github.com/springload/redux-form-exercise/blob/0332cdeac5d9b07baf71caae0fc1c80c675a1ad9/src/components/ExerciseTwoForm.js#L25-L36

### CODE REVIEW

**Comment by vincentaudebert** [comment]
@holloway Please can you review as Thibaud is too busy?

File: src/components/ExerciseTwoForm.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            <fieldset>
+                <legend>Exercise 2</legend>
+                <div>
+                    <div style={{ marginBottom: '10px' }}>
+                        <Field
+                            name="name"
+                            type="text"
+                            component={InputField}
+                            label="Your name:"
+                        />
```
Comment (katie-day) [inline src/components/ExerciseTwoForm.js:53]: Suggestion: add a hint here to use props to format the field using http://redux-form.com/6.0.5/docs/api/Field.md/#props-you-can-pass-to-field-

File: src/components/ExerciseTwoForm.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                Label for input is "Your name:" and label for checkbox is
+                "I agree with these Terms and Conditions?"<br />
+                Please make sure the name is required. It must be displayed in the input as capital
+                letters but it must be stored in lowercase in the state.<br />
+                Also raise a warning if it is not only alphabetic string.<br />
+                Note: There is no need to modify the reducers or the actions.
+            </p>
+            <p>
+                TODO: Implement me in /components/Form/CheckboxField.js &amp;<br />
+                Modify /containers/ExerciseTwoForm.js &amp; /components/Form/InputField.js
```
Comment (katie-day) [inline src/components/ExerciseTwoForm.js]: I don't think anything needs to be changed in `/containers/ExerciseTwoForm.js` maybe remove this from the sentence - could be confusing if it stays there. I believe it's actually `/components/ExerciseTwoForm.js`  that will be modified :)

Comment (vincentaudebert) [reply src/components/ExerciseTwoForm.js]: Good spotting! Sorry about that :)

**Pull request merged**